vcd/datasource_vcd_tm_nsxt_manager.go

```diff
@@ -0,0 +1,67 @@
+package vcd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/go-vcloud-director/v3/govcd"
+	"github.com/vmware/go-vcloud-director/v3/types/v56"
+)
+
+// TODO: TM: validate compatibility with old data source
```

Comment:
Not needed anymore, or do we plan to revisit again in future?

Reply:
Good point. Not needed. Removed this